compiler/rustc_hir_typeck/src/callee.rs

```diff
@@ -523,6 +519,10 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                         ));
 
                         self.select_obligations_where_possible(|_| {});
+                    } else {
+                        if idx == 0 && !self.tcx.is_const_fn_raw(def_id) {
```

Comment:
minor style nit: `else if` over `else { if`

Reply:
Pre-existing, but I'll fix it.